use-cases.md

```diff
@@ -78,6 +78,11 @@ A common event format would allow easier reasoning about the data being
 transited and allow for better introspection of the data.
 
 ### Event Tracing
+An event sent from a source may result in a sequence of additional events
```

Comment:
Nit: for consistency sake, should there be a break line between header and paragraph?

Reply:
Will fix